components/brave_wallet_ui/components/desktop/portfolio-transaction-item/index.tsx

```diff
@@ -298,7 +298,14 @@ const PortfolioTransactionItem = (props: Props) => {
       </TransactionDetailRow>
       <StatusRow>
         <StatusBubble status={transactionDetails.status} />
-        <DetailTextDarkBold>{TransactionStatus[transactionDetails.status]}</DetailTextDarkBold>
```

Comment:
This approach does not work anymore after the migration to use TransactionStatus from mojom directly.